integration_tests/macros/e2e_tests/tests_validation.sql

```diff
@@ -119,8 +119,7 @@
             {% do dimensions_with_problems.append[dimensions] %}
         {% endif %}
     {% endfor %}
-
-    {% if results | length != 2 %}
+    {% if results | length != 3 %}
```

Comment:
I'm not sure why this is hardcoded in the first place, but I added another failed test so this is the new wanted value